src/list-sorting/index.js

```diff
@@ -1,3 +1,15 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+  let haystackLength = haystack.length;
+
+  for (let j = haystackLength - 1; j >= 0; j--) {
```

Comment:
Think you can find a more descriptive variable name than `j`

Reply:
> Think you can find a more descriptive variable name than `j`

Thanks... i will next time